azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/functions/CleanupResources.java

```diff
@@ -79,12 +78,6 @@ public Boolean apply(String id) {
             boolean deploymentDeleteStatus = false;
 
             if (jobDone) {
-               StorageService ss = api.getStorageAccountApi(group).get(storageAccountName);
-               if (ss != null) {
-                  storageAcctDeleteStatus = api.getStorageAccountApi(group).delete(storageAccountName);
-               } else {
-                  storageAcctDeleteStatus = true;
-               }
```

Comment:
@Ladicek Currently, Azure ImageExtension is still a WIP. @jtjk is currently working on this. 

Because the Azure capture image feature requires the new image to be stored in a storage account container, `createImage()` adds the image blob into the same storage account the node is created with. For `CleanupResources`, I temporarily removed the logic that deletes storage account when a node is destroyed because deleting the storage account will also delete all the custom images stored in that storage account. Once ImageExtension is completed, we will be able to query images stored in a storage account and delete a storage account only when no custom image exists.